movie.sh

```diff
@@ -1,6 +1,5 @@
 #!bin/bash
 
-#readonly TEMP_PATH="/var/folders/3c/gjd471t51px36pjlbr7zggmh0000gn/T/imdb"
 readonly TEMP_PATH="$TMPDIR/imdb"
 mkdir $TEMP_PATH      # Making a Folder to Dump Files
```

Comment:
The earlier comment was just an example.
This will still return an error if the directory already exists. There can be many cases where a directory can be created by application and it was not removed. e.g. The user cancelled the execution of the script in between.